molgenis/molgenis
# Search All improvements

Feature:
* Make metadata search results depend on user language
* Make metadata search results case insensitive

Refactor:
* Move SearchAllService to service package
* Move duplicate system package check code to EntityUtils
* Use builder for long parameter list and immutable list attributes in autovalue result classes
* Fix SearchAllServiceTest (getString(DESCRIPTION) was not mocked)
* Move some simple logic to the model
* Add Described interface to model for label / description properties plus match check implementation

Fix:
* Check in yarn lockfile that gets altered during build.

#### Checklist
